packages/orchestrator-ui-components/src/components/WFOForms/UserInputForm.tsx

```diff
@@ -42,6 +41,12 @@ import ConfirmationDialogContext from '../../contexts/ConfirmationDialogProvider
 import AutoFields from './AutoFields';
 import { ValidationError } from '../../types/forms';
 import { ConfirmDialogActions } from '../../contexts/ConfirmationDialogProvider';
+import { useOrchestratorTheme } from '../../hooks';
+
+type JSONSchemaFormProperty = JSONSchema6 & {
+    uniforms: any;
```

Comment:
Cannot be an 'unknown'?

Reply:
It gives a compile error when i change it. This is legacy code. I think we should change it in a sepearate ticket